aperture.go

```diff
@@ -452,10 +468,20 @@ func initTorListener(cfg *config, etcd *clientv3.Client) (*tor.Controller, error
 func createProxy(cfg *config, challenger *LndChallenger,
 	etcdClient *clientv3.Client) (*proxy.Proxy, error) {
 
+	var seed *string
+	if cfg.StaticSecret != nil && cfg.StaticSecret.Enabled {
+		seed = &cfg.StaticSecret.Seed
```

Comment:
We should require the seed to have a minimum length to make sure it provides enough entropy. And we should stretch it to give even more entropy. We use `golang.org/x/crypto/scrypt` for that.
I think something like
```
scryptN = 8192
scryptR = 8
scryptP = 1
```
should provide a good tradeoff between startup speed and bruteforce-ability.